Gems/PhysX/Code/Tests/CharacterControllerTests.cpp

```diff
@@ -159,6 +159,56 @@ namespace PhysX
         }
     }
 
+    TEST_F(PhysXDefaultWorldTest, CharacterController_GamePlayComponent_GravitySetsWhileMoving)
+    {
+        PhysX::CharacterControllerComponent* characterComponent = nullptr;
+        PhysX::CharacterGameplayComponent* gameplayComponent = nullptr;
+        AzFramework::TransformComponent* transform = nullptr;
+
+        float expectedGravityMultiplier = 2.5f;
+
+        // Create character
```

Comment:
If the plan is to write more character controller tests later, this block would be great to split out as a helper function.

Reply:
I had already removed these from my local but forgot to push them up. I'll add this feedback to the notes of the backlog item.